src/components/subscriptions/hooks/licenseManagerHooks.js

```diff
@@ -0,0 +1,168 @@
+import { useState, useEffect } from 'react';
+import qs from 'query-string';
+
+import LicenseManagerApiService from '../../../data/services/LicenseManagerApiService';
+import NewRelicService from '../../../data/services/NewRelicService';
+
+
+export const useSubscriptions = (includeInActive = false) => {
+  const [subscriptions, setSubscriptions] = useState({
+    results: [],
+    count: 0,
+    next: null,
+    previous: null,
+  });
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const [refreshCount, setRefreshCount] = useState(0);
+  const [pageUrl, setPageUrl] = useState(null);
+
+  const refreshSubscriptions = () => setRefreshCount(refreshCount + 1);
+  const fetchNextPage = () => setPageUrl(subscriptions.next);
+  const fetchPreviousPage = () => setPageUrl(subscriptions.previous);
+
+  const fetch = () => {
+    const options = pageUrl ? qs.parseUrl(pageUrl).query : {};
+    setIsLoading(true);
+
+    LicenseManagerApiService.fetchSubscriptions(options)
+      .then((response) => {
+        setIsLoading(false);
+        const subscriptionsData = {
+          ...response.data,
+          ...(
+            includeInActive ? {} : {
+              results: response.data?.results?.filter(subscription => subscription.is_active) || [],
+            }
+          ),
+        };
+        setSubscriptions(subscriptionsData);
+      })
+      .catch((err) => {
+        NewRelicService.logAPIErrorResponse(err);
+        setIsLoading(false);
+        setError(err);
+      });
+  };
+
+  useEffect(() => {
+    fetch();
+  }, [refreshCount, pageUrl]);
+
+
+  return {
+    fetch, subscriptions, isLoading, error, refreshSubscriptions, fetchNextPage, fetchPreviousPage,
+  };
+};
+
+export const useSubscriptionUsersOverview = () => {
+  const [subscriptionUsersOverview, setSubscriptionUsersOverview] = useState({
+    all: 0,
+    active: 0,
+    assigned: 0,
+    deactivated: 0,
+  });
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const [refreshCount, setRefreshCount] = useState(0);
+
+  const [subscriptionUUIDValue, setSubscriptionUUIDValue] = useState();
+
+  const refreshSubscriptionUsersOverview = () => setRefreshCount(refreshCount + 1);
+  const fetch = () => {
+    setIsLoading(true);
+
+    if (!subscriptionUUIDValue) {
+      return;
+    }
+
+    LicenseManagerApiService.fetchSubscriptionUsersOverview(subscriptionUUIDValue)
+      .then((response) => {
+        setIsLoading(false);
+        const subscriptionUsersOverviewData = response.data.reduce((accumulator, currentValue) => ({
+          [currentValue.status]: currentValue.count, ...accumulator,
```

Comment:
Does this properly handle the case where a status may be omitted from the `/overview` response? Trying to think/understand why `undefined` is showing in the tab navigation counts. Does it need the `+` preceding `currentValue.count` like on line 86?